news/2023/2023-06-08-osu-world-cup-2023-mapper-artist-applications.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "osu! World Cup 2023: Mapper & Artist Applications"
+date: 2023-06-08 10:00:00 +0000
+---
+
+Calling out any osu! standard mappers who think they have what it takes – The registrations for mapper and artist contributions for the osu! World Cup 2023 are now open!
+
+![](/wiki/shared/news/banners/OWC_Master_Logo.png)
+
+For those of you who have been living under a particularly comfy rock for the past decade or so, the osu! World Cup is an official, country-based tournament supported by the osu! team, and for the last couple of years, it has used increasingly more bespoke maps made specifically for the tournament; some of which even feature completely new songs made by our very talented [featured artist lineup](https://osu.ppy.sh/beatmaps/artists), which include maps you might know, such as [seatrus - EXXTRA JUDGEMENT](https://osu.ppy.sh/beatmapsets/1889132#osu/3890723), [Yooh - RPG](https://osu.ppy.sh/beatmapsets/1633250#osu/3333745), [II-L feat. amelie xoxo - THE EARTH](https://osu.ppy.sh/beatmapsets/1889101#osu/3941094), [Camellia - OOPARTS](https://osu.ppy.sh/beatmapsets/1312143), and plenty more.
+
+*They love capital letters, don't they? We should probably cut down on the amount of alphabet soup served at the next FA meeting...*
+
+With the increasing amount of content created specifically for the tournament in mind, we have opened a few forms for those who wish to contribute to this ongoing effort.
+
+## Mapper
+
+If you're an osu! mapper who is confident in your mapping abilities, is receptive to feedback and willing to shape their map to the tournament's needs, doesn't mind letter-shaped noodles, and would like to work on a map for the biggest tournament of the year, then head on over to [this registration form](https://docs.google.com/forms/d/e/1FAIpQLSfxQfsFZsSnrcqBN0tUPrlq0OczluC5yH9h1lUZF3x8pgCV9Q/viewform) and put your name out there for the organizers to see! Just make sure you do so before July 2nd, otherwise they might not be able to see anymore or something. You never know.
```

Comment:
```suggestion
If you're an osu! mapper who is confident in your mapping abilities, is receptive to feedback and willing to shape their map to the tournament's needs, doesn't mind letter-shaped noodles, and would like to work on a map for the biggest tournament of the year, then head on over to [this registration form](https://docs.google.com/forms/d/e/1FAIpQLSfxQfsFZsSnrcqBN0tUPrlq0OczluC5yH9h1lUZF3x8pgCV9Q/viewform) and put your name out there for the organisers to see! Just make sure you do so **before July 2nd**, otherwise they may not be seen anymore or something. You never know.
```